src/main/java/org/gradlex/javaecosystem/capabilities/rules/CapabilityDefinitions.java

```diff
@@ -147,12 +153,18 @@ public enum CapabilityDefinitions {
     ),
     C3P0(HIGHEST_VERSION,
         "c3p0:c3p0",
-        "com.mchange:c3p0"
+        "com.mchange:c3p0",
+        "com.mchange:mchange-commons-java"
     ),
     CGLIB(HIGHEST_VERSION,
         "cglib:cglib",
         "cglib:cglib-nodep"
     ),
+    COMMONS_BEANUTILS(HIGHEST_VERSION,
+        "commons-beanutils:commons-beanutils-core",
+        "commons-beanutils:commons-beanutils",
+        "commons-collections:commons-collections"
```

Comment:
Why this?
`commons-collections` and `commons-beanutils:commons-beanutils` do not overlap.

Reply:
As before: https://github.com/scijava/pom-scijava/blob/master/pom.xml#L4502-L4506

Reply:
Removed this